pkg/actions/lua/net/http/request.go

```diff
@@ -0,0 +1,133 @@
+package http
+
+import (
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/Shopify/go-lua"
+	"github.com/treeverse/lakefs/pkg/actions/lua/util"
+)
+
+const defaultRequestTimeout = 30 * time.Second
+
+func Open(l *lua.State) {
+	open := func(l *lua.State) int {
+		lua.NewLibrary(l, httpLibrary)
+		return 1
+	}
+	lua.Require(l, "net/http", open, false)
+	l.Pop(1)
+}
+
+var httpLibrary = []lua.RegistryFunction{
+	{Name: "request", Function: httpRequest},
+}
+
+// httpRequest - perform http request
+//
+//	Accepts arguments (url, body) or table with url, method, body, headers. Value for url is required.
+//		method is by default GET or POST in case body is set.
+//	Returns code, body, headers, status.
```

Comment:
```suggestion
// httpRequest - perform http request
//
//	Accepts arguments (url, body) or table with url, method, body, headers. Value for url is required.
//	Method is by default GET or POST in case body is set.
//	Returns code, body, headers, status.
```

Reply:
Updated the func doc